user/configure-device.rst

```diff
@@ -0,0 +1,132 @@
+disk:
+
+.. code-block:: bash
+
+    VBoxManage convertfromraw --format VDI lede-x86-64-combined-ext4.img lede-x86-64-combined-ext4.vdi
+
+If you encounter an error about ``VERR_ID_INVALID_SIZE``, you need to
+pad the image with the following command:
+
+.. code-block:: bash
+
+    dd if=lede-x86-64-combined-ext4.img of=lede-padded.img bs=128000 conv=sync
+
+And then try to convert the padded image using the previous command.
+
+Next, open up VirtualBox and create a VM. Load up the VDI and then start
+the machine. If done correctly, it should boot to a GRUB menu and then
+proceed with initialization. When the text stops scrolling, simply press
+Enter to activate the terminal.
+
+.. note::
+
+    Continue with the procedures in `this guide <https://lede-project.org/docs/user-guide/virtualbox-vm>`_. It's required to
+    enable SSH access and connect the VM to the internet.
+
+3. Install openwisp-config on your LEDE instance
+------------------------------------------------
+
+Installation
+~~~~~~~~~~~~
+
+.. note::
+
+    An installation guide is also available
+    `here <https://github.com/openwisp/openwisp-config#install-precompiled-package>`__.
+
+Run the following commands on your device:
+
+.. code-block:: bash
+
+    opkg update
+    opkg install <URL>
+
+Replace ``<URL>`` with the link to one of the latest build, available
+`here <http://downloads.openwisp.org/openwisp-config/latest/>`__. For
+this guide, let's choose the ``openssl`` package, but any of them should
+work the same.
+
+Configuration
+~~~~~~~~~~~~~
+
+Once it's installed, we need to configure it to connect to our OpenWISP2
+controller. To do that, edit the config file located at
+``/etc/config/openwisp``.
+
+You will see the default config file, something like the following:
+
+::
+
+    # For more information about these config options please see the README
+    # or https://github.com/openwisp/openwisp-config#configuration-options
+
+    config controller 'http'
+        #option url 'https://openwisp2.mynetwork.com'
+        #option interval '120'
+        #option verify_ssl '1'
+        #option shared_secret ''
+        #option consistent_key '1'
+        #option mac_interface 'eth0'
+        #option merge_config '1'
+        #option test_config '1'
+        #option test_script '/usr/sbin/mytest'
+        option uuid ''
+        option key ''
+        list unmanaged 'system.@led'
+        list unmanaged 'network.loopback'
+        list unmanaged 'network.@switch'
+        list unmanaged 'network.@switch_vlan'
+        # curl options
+        #option connect_timeout '15'
+        #option max_time '30'
+        #option capath '/etc/ssl/certs'
+
+Uncomment and change the following fields:
+
+-  ``url``: The hostname of your OpenWISP2 controller
+-  ``verify_ssl``: Set to ``0`` if your controller's SSL certificate is
+   self-signed
+-  ``shared_secret``: Get it from OpenWISP2's admin panel, located at
+   the Group settings
```

Comment:
it's located in the organization settings, if you can make this passage more precise you will save time to a lot of users, you could also add the precise URL path without the domain